CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
+## 0.15.0 - 2024-11-08
+
+This release makes scale-decode `no_std` which is now using `core::error::Error` instead of `std::error::Error` as it was using before behind
```

Comment:
```suggestion
This release makes scale-decode entirely `no_std` which is now using `core::error::Error` instead of `std::error::Error` as it was using before behind
```

Just because it was already no_std compatible :)